hooks/ROT.js

```diff
@@ -36,28 +36,23 @@ module.exports = function (parser, input) {
   const { id, sentence, parts, tags } = input
 
   if (String(parts[1]).toUpperCase() !== 'A') {
-    // Don't parse this sentence as it's void.
-    return Promise.reject(new Error('Not parsing sentence as data is not valid)'))
+    return null
```

Comment:
I have changed the behavior here because I think this is more coherent with what we do in other hooks. Happy to discuss / change everything to throw instead of returning null.